test_everything_inspect.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright 2017 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License");
+
+. $(dirname $0)/common.sh
+PARENT_DIR="RUN_EVERY_BENCHMARK"
+LOGS_1="./${PARENT_DIR}/RUNDIR-*/fuzz-0.log"
+LOGS_2="./${PARENT_DIR}/RUNDIR-*/log"
+
+less $LOGS_1
```

Comment:
This looks weird as it sounds like a fully manual step. Maybe don't include this file.